src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1906,6 +1906,9 @@ namespace OpenRCT2::RCT2
             auto animObjects = GetLegacyPeepAnimationObjects();
             AppendRequiredObjects(objectList, ObjectType::peepAnimations, animObjects);
 
+            auto climateObjId = GetClimateObjectIdFromLegacyClimateType(RCT12::ClimateType{ _s6.Climate });
```

Comment:
Why not change the member type _s6.Climate and _s4.Climate to the enum in the first place?